.circleci/config.yml

```diff
@@ -602,6 +602,28 @@ jobs:
             echo "JS_ENGINES = [JSC_ENGINE]" >> ~/emsdk/.emscripten
       - run-tests:
           test_targets: "core0.test_hello_world"
+  test-spidermonkey:
+    executor: linux-python
+    steps:
+      - checkout
+      - run:
+          name: submodule update
+          command: git submodule update --init
+      - pip-install
+      - build
+      - run:
+          name: install spidermonkey
+          command: |
+            cd $HOME
+            wget https://nodejs.org/dist/v18.12.0/node-v18.12.0-linux-x64.tar.xz
```

Comment:
"Install SpiderMonkey" section begins with installing Node.js :thinking: 

Mostly joking, but perhaps we could have an "install jsvu" section?

Reply:
One reason to keep it like this is that the `export PATH..` statement (which gives access to both jsvu and npm) doesn't outlive the job.